float32MultiArray joint_positions
float32MultiArray joint_velocities
float32MultiArray joint_currents
float32MultiArray joint_torques

EEpos left_ee_pos
EEpos right_ee_pos
EEpos torso_ee_pos

FTsensor left_ft_sensor
FTsensor right_ft_sensor

float32MultiArray center_of_mass

float32 mobile_angular_velocity
float32 mobile_linear_velocity

bool is_initialized
bool is_stopped
bool is_torso_following
bool is_left_following
bool is_right_following